suite-native/device-manager/src/components/DeviceManagerContent.tsx

```diff
@@ -1,90 +1,98 @@
-import { useSelector } from 'react-redux';
-import { useMemo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useState } from 'react';
 
-import { A } from '@mobily/ts-belt';
-import { useNavigation } from '@react-navigation/native';
-
-import { analytics, EventType } from '@suite-native/analytics';
-import { Button, Text, VStack } from '@suite-native/atoms';
+import { HStack, VStack } from '@suite-native/atoms';
 import {
-    selectDevices,
+    PORTFOLIO_TRACKER_DEVICE_ID,
+    selectDevice,
+    selectDeviceThunk,
+    selectIsDeviceDiscoveryActive,
     selectIsPortfolioTrackerDevice,
-    selectDeviceId,
-    selectIsNoPhysicalDeviceConnected,
 } from '@suite-common/wallet-core';
-import {
-    ConnectDeviceStackRoutes,
-    RootStackParamList,
-    RootStackRoutes,
-    StackToStackCompositeNavigationProps,
-} from '@suite-native/navigation';
-import { Translation } from '@suite-native/intl';
 import { FeatureFlag, useFeatureFlag } from '@suite-native/feature-flags';
+import { EventType, analytics } from '@suite-native/analytics';
+import { TrezorDevice } from '@suite-common/suite-types';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 
+import { AddHiddenWalletButton } from './AddHiddenWalletButton';
+import { DeviceList } from './DeviceList';
+import { DeviceInfoButton } from './DeviceInfoButton';
 import { DeviceManagerModal } from './DeviceManagerModal';
-import { DeviceItem } from './DeviceItem';
-import { DeviceControlButtons } from './DeviceControlButtons';
+import { DevicesToggleButton } from './DevicesToggleButton';
+import { WalletList } from './WalletList';
 import { useDeviceManager } from '../hooks/useDeviceManager';
-import { AddHiddenWalletButton } from './AddHiddenWalletButton';
 
-type NavigationProp = StackToStackCompositeNavigationProps<
-    RootStackParamList,
-    RootStackRoutes.AppTabs,
-    RootStackParamList
->;
+const deviceButtonsStyle = prepareNativeStyle(utils => ({
+    width: '100%',
+    paddingHorizontal: utils.spacings.medium,
+    paddingBottom: utils.spacings.medium,
+}));
 
 export const DeviceManagerContent = () => {
-    const navigation = useNavigation<NavigationProp>();
+    const { applyStyle } = useNativeStyles();
+    const [isChangeDeviceRequested, setIsChangeDeviceRequested] = useState(false);
```

Comment:
nit: How about this naming? Feel free to resolve if you think the currenct name is better

```suggestion
    const [isDeviceListDisplayed, setIsDeviceListDisplayed] = useState(false);
```

Reply:
the list might be forced to be visible in case it is portfolio tracker and Peta suggested not to use the naming you propose. I'll keep this one here